tests/unit/sort-step-form.spec.ts

```diff
@@ -60,6 +60,28 @@ describe('Sort Step Form', () => {
     ]);
   });
 
+  it('should not suggest a column to be sorted if no column is selected', function() {
+    const wrapper = runner.shallowMount({});
+    expect(wrapper.find('listwidget-stub').props().value).toEqual([{ column: '', order: 'asc' }]);
```

Comment:
```suggestion
    expect(wrapper.find(Listwidget).props().value).toEqual([{ column: '', order: 'asc' }]);
```
If you import `Listwidget` in the test you can use this cool syntax.
(This syntax is used on the "common tips" of Vue test utils: https://vue-test-utils.vuejs.org/guides/common-tips.html)

Reply:
Thank you for the comment.

I'm not very convinced to implement that for several reasons:

- I find it kind of a waste to import a full component just to make the naming a bit fancier (but not that much more understandable than the same name with just a '-stub' suffix
- In my opinion we would be less explicit using the component name, as we do manipulate stubs, not real components
- At the end we are talking about details and I would be OK to change, but in that case I would prefer to do it in a separate PR, in all test files to keep a coherence